moviepy/video/VideoClip.py

```diff
@@ -11,7 +11,7 @@
 
 import numpy as np
 import proglog
-from imageio import imread, imsave
+from imageio.v3 import imread, imwrite
```

Comment:
While not strictly necessary, it is recommended to import Imageio as `import imageio.v3 as iio`. This gives you access to the full API and reads nicer than only importing the methods used.

You would then consume it as `iio,imwrite` and `iio.imread` respectively.

Reply:
Thanks for pointing out. I'll push a new commit